mycli/main.py

```diff
@@ -170,6 +173,12 @@ def execute_from_file(self, arg, **_):
 
         return self.sqlexecute.run(query)
 
+    def execute_system_command(self, arg, **_):
+          if not arg:
+              message = 'Missing required argument: command.'
+              yield(None, None, None, message)
+          yield(None, None, None, subprocess.call(arg, shell=True))
+
```

Comment:
@amjith Hmm. Cool! Thanks for clarifying that to me! I'll move the function to iocommands.py. :)